package.json

```diff
@@ -17,7 +17,7 @@
     "build": "babel src -d lib --source-maps",
     "start": "babel src -w -d lib --source-maps",
     "clean": "rimraf lib",
-    "prepublish": "npm run build"
+    "prepublish": "npm run build; git rev-parse HEAD > version.txt"
```

Comment:
hmm, I've been trying to avoid putting things in package.json scripts that need unix or git installed as this gets run automatically by npm at times, so this will cause breakage for people on windows.